SAMLPortal/Misc/Helpers.cs

```diff
@@ -131,10 +136,74 @@
 			return b64Cert;
 		}
 
+		/// <summary>
+		/// Convert a base64 encoded X509 certificate to a X509Certificate 2 object
+		/// </summary>
+		/// <param name="b64Cert">base64 encoded string of the certificate</param>
+		/// <returns>X509Certificate2 object</returns>
 		public static X509Certificate2 StringToX509Certificate2(string b64Cert)
 		{
 			var decodedCert = Convert.FromBase64String(b64Cert);
 			return new X509Certificate2(decodedCert);
 		}
+
+		/// <summary>
+		/// Write environment variable to a file in the format VARIABLE=VALUE
+		/// </summary>
+		/// <param name="filePath">Path of the file to write to</param>
+		/// <param name="key">Name of the environment variable</param>
+		/// <param name="value">Value of the environment variable</param>
+		public static void WriteEnvVariableToFile(string filePath, string key, string value)
+		{
+			if (!key.Contains(" ") && !value.Contains(" "))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'value' before using it.](https://app.codacy.com/manual/zegorax/SAMLPortal/pullRequest?prid=5343548)